agences-regionales-sante/hauts-de-france/2020-03-18.yaml

```diff
@@ -0,0 +1,33 @@
+date: "2020-03-18"
+source:
+  nom: ARS Hauts-de-France
+  url: https://www.hauts-de-france.ars.sante.fr/coronavirus-covid-19-point-de-situation-dans-les-hauts-de-france
+  archive: https://web.archive.org/web/20200319001345/https://www.hauts-de-france.ars.sante.fr/coronavirus-covid-19-point-de-situation-dans-les-hauts-de-france
```

Comment:
L'archive pointe vers les données du 17/03. Peux-tu créer une nouvelle archive (j'ai noté comment faire dans le readme du projet)

Reply:
Tu es sûr ? De mon côté, le lien de l'archive renvoie bien vers le point de situation du 18/03

Reply:
Yes c'est bien le 18 mars.

Dans le chapeau :

> Données au 18 mars 2020 à 15h.

Reply:
Maeculpa, soit archive.org bug, soit -je- bug :P